SQL/0000-00-00-schema.sql

```diff
@@ -368,6 +370,24 @@ CREATE TABLE `test_names` (
   CONSTRAINT `FK_test_names_1` FOREIGN KEY (`Sub_group`) REFERENCES `test_subgroups` (`ID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `test_json` (
+  `TestJsonID` varchar(64) NOT NULL,
+  `JsonData` TEXT NOT NULL,
+  CONSTRAINT `PK_test_json` PRIMARY KEY (`TestJsonID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `test_names_json_rel` (
```

Comment:
given our current convention this table name does not follow the `table1_table2_rel` name

Reply:
no it does not .. but I think it's an acceptable short form because `test_names_test_json_rel` seems redundant so i removed the repeat of `test`. this can easily be changed to properly conform to the convention